crowbar_framework/app/controllers/updater_controller.rb

```diff
@@ -15,6 +15,17 @@
 #
 
 class UpdaterController < BarclampController
+  before_filter :initialize_service
+  before_filter :controller_to_barclamp
+
+  def controller_to_barclamp
```

Comment:
You don't need that. You are in the `UpdaterController` and it initializes the `UpdaterService` within `initialize_service`